### PR TITLE
TASK: Update Psalm baseline

### PR DESCRIPTION
**Pull request by albe**

### CODE REVIEW

File: psalm-baseline.xml [hunk 1]
```diff
@@ -53,6 +53,25 @@
     <PossiblyNullPropertyAssignmentValue occurrences="1">
       <code>null</code>
     </PossiblyNullPropertyAssignmentValue>
+    <UndefinedClass occurrences="2">
```
Comment (albe) [inline psalm-baseline.xml]: Not sure where these come from... they definitely weren't in the psalm errors before. Maybe difference local run vs. travis environment?

File: psalm-baseline.xml [hunk 2]
```diff
@@ -3005,6 +3042,16 @@
       <code>$validator-&gt;setValidatedInstancesContainer</code>
     </NoInterfaceProperties>
   </file>
+  <file src="Packages/Framework/Neos.Flow/Classes/Validation/Validator/CollectionValidator.php">
```
Comment (albe) [inline psalm-baseline.xml]: These two are the errors from #1767

File: psalm-baseline.xml (hunk 1 above)
Comment (kdambekalns) [reply psalm-baseline.xml]: Maybe because I don't have Redis installed? Pretty stupid check, then… :)

But no… I have ext-redis installed. 🤷‍♂️

**Comment by albe** [comment]
https://travis-ci.org/neos/flow-development-collection/jobs/586487624

travis too as well as another one... don't get that on my machine though

**Comment by albe** [comment]
Ok, tried with a fresh install of the development collection and that seems to have to do with it

**Comment by albe** [comment]
Green now. Okay to merge @kdambekalns ?

**Pull request merged**